Makefile

```diff
@@ -262,9 +275,9 @@ audit: sign audit-licenses
 
 publish: audit publish-bin publish-pip publish-src publish-jars
 
-all: clean test audit
+all: clean system-test test audit
```

Comment:
Did not quite understood the docs [here](http://www.gnu.org/software/make/manual/make.html#Prerequisite-Types)... but see what you figure out.